src/features/enrollments/CourseEnrollmentList.jsx

```diff
@@ -1,35 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { DataTable } from '@edx/paragon';
+import { DataTable, DropdownFilter, TextFilter } from '@edx/paragon';
 import useRecords from './useRecords';
 
+import { getCohortFilterOptions } from '../../utils/forms';
+
 function courseEnrollments(courseKey) {
-  return enrollment => enrollment.offering.courseKey === courseKey;
+  return (enrollment) => enrollment.offering.courseKey === courseKey;
 }
 
 function courseCompletions(courseKey) {
-  return enrollment => (
+  return (enrollment) => (
     enrollment.isComplete && enrollment.offering.courseKey === courseKey
   );
 }
 
-export default function CourseEnrollmentList({ offerings }) {
-  const [enrollments] = useRecords();
+export default function CourseEnrollmentList({ offerings, cohorts }) {
+  const [enrollments, enrollmentsStatus] = useRecords();
 
-  const data = offerings.map(offering => ({
-    title: offering.title,
-    courseKey: offering.courseKey,
-    enrollments: enrollments.filter(courseEnrollments(offering.courseKey)).length,
-    completions: enrollments.filter(courseCompletions(offering.courseKey)).length,
+  const data = offerings.map((offering) => ({
+    title: offering.details.title,
+    courseKey: offering.details.courseKey,
+    cohort: offering.cohort,
```

Comment:
Ah, this needs to be a list as well, since an offering can be available to multiple cohorts.